tests/package.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+npm pack . --silent
+mv node-binance-api*.tgz ./tests/package-test/
+cd ./tests/package-test
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)